internal/pkg/server/fleet_integration_test.go

```diff
@@ -1370,3 +1371,148 @@ func Test_SmokeTest_CheckinPollShutdown(t *testing.T) {
+	res.Body.Close()
+	var obj map[string]interface{}
+	err = json.Unmarshal(p, &obj)
+	require.NoError(t, err)
+
+	at, ok := obj["ack_token"]
+	require.True(t, ok, "expected ack_token in response")
+	_, ok = at.(string)
+	require.True(t, ok, "ack_token is not a string")
+
+	actionsRaw, ok := obj["actions"]
+	require.True(t, ok, "expected actions is missing")
+	actions, ok := actionsRaw.([]interface{})
+	require.True(t, ok, "expected actions to be an array")
+	require.Greater(t, len(actions), 0, "expected at least 1 action")
+	action, ok := actions[0].(map[string]interface{})
+	require.True(t, ok, "expected action to be an object")
+	aIDRaw, ok := action["id"]
+	require.True(t, ok, "expected action id attribute missing")
+	aID, ok := aIDRaw.(string)
+	require.True(t, ok, "expected action id to be string")
+	aAgentIDRaw, ok := action["agent_id"]
+	require.True(t, ok, "expected action agent_id attribute missing")
+	aAgentID, ok := aAgentIDRaw.(string)
+	require.True(t, ok, "expected action agent_id to be string")
+	require.Equal(t, id, aAgentID)
+
+	body := fmt.Sprintf(`{
+	    "events": [{
+		"action_id": "%s",
+		"agent_id": "%s",
+		"message": "test-message",
+		"type": "ACTION_RESULT",
+		"subtype": "ACKNOWLEDGED"
+	    }]
+	}`, aID, id)
+	t.Logf("Fake an ack for action %s for agent %s", aID, id)
+	req, err = http.NewRequestWithContext(ctx, "POST", srv.baseURL()+"/api/fleet/agents/"+id+"/acks", strings.NewReader(body))
+	require.NoError(t, err)
+	req.Header.Set("Authorization", "ApiKey "+key)
+	req.Header.Set("Content-Type", "application/json")
+	res, err = cli.Do(req)
+	require.NoError(t, err)
+
+	require.Equal(t, http.StatusOK, res.StatusCode)
+	t.Log("Ack successful, verify body")
+	p, _ = io.ReadAll(res.Body)
+	res.Body.Close()
+	var ackObj map[string]interface{}
+	err = json.Unmarshal(p, &ackObj)
+	require.NoError(t, err)
+
+	// NOTE the checkin response will only have the errors attribute if it's set to true in the response.
+	// When decoding to a (typed) struct, the default will implicitly be false if it's missing
+	_, ok = ackObj["errors"]
+	require.Falsef(t, ok, "expected response to have no errors attribute, errors are present: %+v", ackObj)
+
+	// Update agent doc to have output key == ""
+	agent, err := dl.FindAgent(ctx, srv.bulker, dl.QueryAgentByID, dl.FieldID, id)
+	require.NoError(t, err)
+	outputNames := make([]string, 0, len(agent.Outputs))
+	for name := range agent.Outputs {
+		outputNames = append(outputNames, name)
+	}
+	require.Len(t, outputNames, 1)
+	p = []byte(fmt.Sprintf(`{"script":{"lang": "painless", "source": "ctx._source['outputs'][params.output].api_key == ''; ctx._source['outputs'][params.output].api_key_id == '';", "params": {"output": "%s"}}}`, outputNames[0]))
+	t.Logf("Attempting to remove api_key attribute from: %s, body: %s", id, string(p))
+	err = srv.bulker.Update(
+		ctx,
+		dl.FleetAgents,
+		id,
+		p,
+		bulk.WithRefresh(),
+		bulk.WithRetryOnConflict(3),
+	)
+	require.NoError(t, err)
```

Comment:
This was failing when I ran the integration tests locally with
```
    fleet_integration_test.go:1478: Attempting to remove api_key attribute from: 8abbbaf3-cfec-41ae-a086-540cc8604885, body: {"script":{"lang": "painless", "source": "ctx._source['outputs'][params.output].api_key == ''; ctx._source['outputs'][params.output].api_key_id == '';", "params": {"output": "default"}}}
    fleet_integration_test.go:1487:
        	Error Trace:	/Users/laterman/git/fleet-server/internal/pkg/server/fleet_integration_test.go:1487
        	Error:      	Received unexpected error:
        	            	elastic fail 400: illegal_argument_exception: failed to execute script: script_exception: compile error
        	Test:       	Test_SmokeTest_Verify_v85Migrate
--- FAIL: Test_SmokeTest_Verify_v85Migrate (3.98s)
```
I may need some help figuring out what the actual error is